exporter/exporterbatcher/sizer_type.go

```diff
@@ -0,0 +1,61 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package exporterbatcher // import "go.opentelemetry.io/collector/exporter/exporterbatcher"
+
+import (
+	"fmt"
+)
+
+type SizerType struct {
+	string
+}
```

Comment:
No need to make it struct
```suggestion
type SizerType string
```